src/components/main-panel/main-panel.tsx

```diff
@@ -38,11 +44,14 @@ export const MainPanel = () => {
         focusElement,
         focusedElement,
         updateOrderedIdentifiers,
+        removeFocusedElement,
     } = useBuilder();
-
+    const { isDirty } = focusedElement || {};
     const finalMode = focusedElement ? true : showSidePanel;
     const renderMode = finalMode ? "minimised" : "expanded";
     const items: (UniqueIdentifier | { id: UniqueIdentifier })[] = [];
+    const { showModal, discardChanges } = useModal();
+    const { showToast } = useDisplay();
```

Comment:
remove unused hook, be sure to remove `useDisplay` import as well

Reply:
updated in the latest changes